docs/guides/tutorials/rest_apis_with_fastapi.rst

```diff
@@ -872,30 +813,32 @@ drop this query into it:
         }
     ) {name, address, schedule, host: {name}};
 
-Run ``edgedb-py --file`` to generate a function from that query.
+Run ``edgedb-py`` to generate a function from that query.
 
 Create a file in ``app`` named ``events.py`` and open it in your editor. It's
 time to code up the endpoint to use that freshly generated query.
 
+.. lint-off
+
 .. code-block:: python
 
     # app/events.py
     from __future__ import annotations
 
     from http import HTTPStatus
-    from typing import Iterable, Optional
+    from typing import List
 
     import edgedb
     from fastapi import APIRouter, HTTPException, Query
     from pydantic import BaseModel
 
-    import generated_async_edgeql as db_queries
+    from .queries import (
+        create_event_async_edgeql,
```

Comment:
```suggestion
        create_event_async_edgeql as create_event_qry,
```

Potentially you can alias long module names to make them look prettier

Reply:
I'll circle back to the example and implement this and then update the guide. Thanks for the tip, @1st1!

Reply:
Examples PR is here: https://github.com/edgedb/edgedb-examples/pull/104